src/BiomeDef.cpp

```diff
@@ -222,3 +222,130 @@ bool IsBiomeCold(EMCSBiome a_Biome)
 
 
 
+
+int GetSnowStartHeight(EMCSBiome a_Biome)
+{
+	switch (a_Biome)
+	{
+		case biIcePlainsSpikes:
+		case biIcePlains:
+		case biIceMountains:
+		case biFrozenRiver:
+		case biColdBeach:
+		case biColdTaiga:
+		case biColdTaigaHills:
+		case biColdTaigaM:
+		{
+			// Always snow
+			return -1;
```

Comment:
You may want to return 0, and return an `unsigned` from this function, so that it compares with `HEIGHTTYPE` without clang/gcc warnings.